src/block.js

```diff
@@ -464,7 +464,9 @@ Object.assign(Block.prototype, SimpleBlock.fn, require('./block-validations'), {
 
   select: function(selected) {
     this.el.classList.toggle("st-block--is-selected", selected);
-  }
+  },
+
+  split: function() {}
```

Comment:
Does this need to be implemented differently for each kind of block?

Reply:
If other blocks include a text block that can be split then yes. Otherwise it'll add the pasted content within the current selection and beyond.